### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by OlehHatsko**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by tetyanakutsko — APPROVED** [review]

File: src/pages/userPage/EditUserPage/EditUserPage.tsx [hunk 1]
```diff
@@ -47,6 +47,7 @@ export default function () {
   const secondPatern = /^[a-zA-Zа-яА-ЯІіЄєЇїҐґ'"\(\).`]{0,50}((\s+|-)[a-zA-Zа-яА-ЯІіЄєЇїҐґ'"\(\).`]{0,50})*$/;
   const message = shouldContain("тільки літери");
   const [form] = Form.useForm();
+  const MAX_AGE = 100;
```
Comment (Lunaflumen) [inline src/pages/userPage/EditUserPage/EditUserPage.tsx:50]: Why didn`t use lover camel case?

**Review by Lunaflumen — CHANGES_REQUESTED** [review]

File: src/pages/userPage/EditUserPage/EditUserPage.tsx (hunk 1 above)
Comment (voroniak) [reply src/pages/userPage/EditUserPage/EditUserPage.tsx:50]: I guess it's const variable

Comment (OlehHatsko) [reply src/pages/userPage/EditUserPage/EditUserPage.tsx:50]: Because it's constant value

Comment (Lunaflumen) [reply src/pages/userPage/EditUserPage/EditUserPage.tsx:50]: Ok, it`s my mistake (didn`t know about this rule of constants naming)

**Review by Lunaflumen — APPROVED** [review]

**Review by MykhailoKlapchuk — APPROVED** [review]

**Pull request merged**